internal/providersdkv2/data_source_vault_cluster.go

```diff
@@ -276,6 +291,21 @@ If a project is not configured in the HCP Provider config block, the oldest proj
 							Type:        schema.TypeString,
 							Computed:    true,
 						},
+						"newrelic_account_id": {
+							Description: "NewRelic Account ID for streaming audit logs",
+							Type:        schema.TypeString,
+							Computed:    true,
+						},
+						"newrelic_license_key": {
+							Description: "NewRelic license key for streaming audit logs",
+							Type:        schema.TypeString,
+							Computed:    true,
+						},
+						"newrelic_region": {
+							Description: "NewRelic region for streaming audit logs",
+							Type:        schema.TypeString,
```

Comment:
we don't do any client-side validation on these and have them all as strings, which I think is fine even tho on the frontend we have enums. But can we highlight in the description that it needs to be either EU or US?

Reply:
Will make this change.